### DISCUSSION TITLE
消除错误 Since Maven 3.8.1 http repositories are blocked — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#263] [Announcements]
# 消除错误 Since Maven 3.8.1 http repositories are blocked — 码志

今天给一个老项目的 pom.xml 里加了一个依赖项后，IDEA 突然报错 Since Maven 3.8.1 http repositories are blocked。

https://mazhuang.org/fragment/maven-http-repositories-are-blocked/

**Comment by flamingstar** [comment]
居然搜到了壮哥的文章，点个赞

**Reply by mzlogin** [reply]
哈哈哈哈 这么巧……睡这么晚
